src/commands/until_front_of_line.yml

```diff
@@ -72,32 +72,51 @@ steps:
           fi
         }
 
+        fetch_active_workflows(){
+          for workflow in `jq -r ".[] | .workflows.workflow_id" /tmp/jobstatus.json | uniq`
+          do
+            workflow_file=/tmp/workflow-${workflow}.json
+            if [ ! -z $TESTING_MOCK_WORKFLOW_RESPONSES ] && [ -f $TESTING_MOCK_WORKFLOW_RESPONSES/${workflow}.json ]; then
+              echo "Using test mock workflow response"
+              cat $TESTING_MOCK_WORKFLOW_RESPONSES/${workflow}.json > ${workflow_file}
+            else
+              curl -f -s "https://circleci.com/api/v2/workflow/${workflow}?circle-token=${CIRCLECI_API_KEY}" > ${workflow_file}
```

Comment:
I'm not sure that this variant with **CIRCLECI_API_KEY** will work as well, because CircleCi API v1.1 and v2 use different tokens. At least for me, I had to create an additional token for v2 which doesn't work for v1.1 and vice verse. 
Please correct me if I wrong.